podcasts/TranscriptViewController.swift

```diff
@@ -1,7 +1,7 @@
 import UIKit
 import PocketCastsUtils
 
-class TranscriptsViewController: PlayerItemViewController {
+class TranscriptViewController: PlayerItemViewController {
```

Comment:
I've updated the name of this VC to reflect what it really is: a single transcript (not Transcripts, in the plural)